### PR TITLE
Simple tool for adding all org users with 2fa to a team

### PR DESCRIPTION
**Pull request by timgent**
Basic starting point, but does the job we need it to do for the minute

### CODE REVIEW

**Comment by allandegnan** [comment]
Spoke with Tim, apparently there isn't something else out there that does what we want it to do and this is fairly trivial and the only real test would be to a) try it, or b) stub github api that's probably a moving target...

so yeah, lgtm.

File: github-org-team-creator.go [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+		org := c.Args().Get(0)
+		team := c.Args().Get(1)
+		ts := oauth2.StaticTokenSource(
+			&oauth2.Token{AccessToken: accessToken},
+		)
+		tc := oauth2.NewClient(oauth2.NoContext, ts)
+		githubClient := github.NewClient(tc)
+		teamId := getTeamId(org, team, githubClient)
+		if teamId == 0 {
+			println("Creating team...")
```
Comment (gambol99) [inline github-org-team-creator.go]: wheres println coming from? ... i don't see a method anywhere

Comment (timgent) [reply github-org-team-creator.go]: Not sure... I just typed some scala and it worked :D Assumed it was a built in golang method?

Is there a recommended alternative I should use?

Comment (marcinc) [inline github-org-team-creator.go]: use `fmt.Println()` instead (also in other places)

File: github-org-team-creator.go [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+	println("Github team " + team + " doesn't exist")
+	return 0
+}
+
+func handleError(err error) {
+	if err != nil {
+		log.Fatal(err)
+		os.Exit(1)
+	}
+}
```
Comment (marcinc) [inline github-org-team-creator.go]: NL missing

Comment (timgent) [reply github-org-team-creator.go]: NL?

Comment (timgent) [reply github-org-team-creator.go]: Oh, new line?

File: github-org-team-creator.go (hunk 1 above)
Comment (marcinc) [reply github-org-team-creator.go]: yes - `fmt.Println()`

Comment (marcinc) [reply github-org-team-creator.go]: `println` is not guaranteed to stay in the language: https://golang.org/ref/spec#Bootstrapping

Comment (timgent) [reply github-org-team-creator.go]: Changed to fmt.Println()

Comment (timgent) [reply github-org-team-creator.go]: Done

File: github-org-team-creator.go (hunk 2 above)
Comment (timgent) [reply github-org-team-creator.go]: Done

**Comment by marcinc** [comment]
Is this related to this issue https://github.com/UKHomeOffice/central-devteam-board/issues/33?

**Comment by timgent** [comment]
@marcinc It was done more for this one:
https://github.com/UKHomeOffice/central-devteam-board/issues/29
Wanted to give people read only access to our kanban boards and write access to the BAU board. Otherwise we tell people to go to them but they all get access denied! This was the easiest way I could see to do it, though it seems weird you can't easily give everyone in an org access to a private github repo

Just a quick and dirty fix to get us going, know there are tickets in the central team backlogs to sort github user management properly

**Comment by marcinc** [comment]
Cool. LGTM.

**Pull request merged**